src/controllers/merchant.controller.ts

```
@@ -3,11 +3,19 @@ import { Request } from "express-serve-static-core";
 import { validateWebhookUrl } from "../validators/webhook.validators";
 import crypto from "crypto";
 import { Merchant, MerchantWebhook } from "../interfaces/webhook.interfaces";
+import { CustomRequest } from "../middlewares/merchantAuth";
 import { MerchantAuthService } from "../services/merchant.service";
 import { WebhookService } from "../services/webhook.service";
+import { validateWebhookUrl } from "../validators/webhook.validators";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'validateWebhookUrl'. Consider to delete it or rename it.

'validateWebhookUrl' is defined here:


(lint/suspicious/noRedeclare)